juriscraper/opinions/united_states/state/lactapp_4.py

```diff
@@ -0,0 +1,207 @@
+Court Short Name: La. Ct. App. 4th Cir.
+Author: Luis-manzur
+History:
+  2025-04-22: Created by Luis-manzur
+"""
+
+import re
+from datetime import date, datetime
+
+from juriscraper.AbstractSite import logger
+from juriscraper.lib.date_utils import unique_year_month
+from juriscraper.lib.string_utils import titlecase
+from juriscraper.lib.type_utils import OpinionType
+from juriscraper.OpinionSiteLinear import OpinionSiteLinear
+
+
+class Site(OpinionSiteLinear):
+    first_opinion_date = datetime(2019, 7, 17)
+    days_interval = 28  # ensure a tick for each month
+    date_regex = re.compile(r"\d{2}/\d{2}/\d{4}")
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.court_id = self.__module__
+        self.url = "https://www.la4th.org/Default.aspx"
+        self.search_is_configured = False
+        self.target_date = date.today()
+        self.make_backscrape_iterable(kwargs)
+        self.status = "Published"
+        self.parameters = {}
+
+    def _process_html(self):
+        """Process the HTML to extract case details."""
+
+        # XPath for the opinion results
+        opinion_results_xpath = "//div[contains(@class, 'opinion-result')]"
+        results = self.html.xpath(opinion_results_xpath)
+
+        # Log the number of results found
+        logger.info("Found %d results on the page.", len(results))
+
+        self.cases = []
+        for result in results:
+            docket = result.xpath(".//strong/text()")[0]
+            name = result.xpath(".//p[not(strong)]/text()")[0]
+            decree = result.xpath(
+                ".//p[strong[contains(text(), 'Decree')]]/text()"
+            )[0]
+            date = result.xpath(
+                ".//p[strong[contains(text(), 'Opinion Date')]]/text()"
+            )[0]
+            download_url = result.xpath(
+                ".//p/a[contains(text(), 'View Document')]/@href"
+            )[0]
+
+            # Clean and structure the extracted data
+            case = {
+                "docket": docket if docket else None,
+                "name": titlecase(name) if name else None,
+                "disposition": titlecase(decree) if decree else None,
+                "date": date if date else None,
+                "url": download_url if download_url else None,
+            }
+
+            self.cases.append(case)
+
+    def update_date_filters(self) -> None:
+        """Set year and month values from `self.target_date`
+        into self.parameters for POST use
+        """
+        logger.info(
+            "Scraping for year: %s - month: %s",
+            self.target_date.year,
+            self.target_date.month,
+        )
+
+        month_name = self.target_date.strftime("%B")
+
+        self.parameters.update(
+            {
+                "ctl00$Main$ddlOpMonth": month_name,
+                "ctl00$Main$ddlOpYear": str(self.target_date.year),
+                "__EVENTTARGET": "ctl00$Main$btnOpMonthYearSearch",
```

Comment:
EVENT TARGET doesnt appear to actually change. so just put it in the self.parameters dict at the start and remove it from here